win32ss/user/ntuser/menu.c

```diff
@@ -6041,16 +6030,17 @@ NtUserGetMenuBarInfo(
    if (!NT_SUCCESS(Status))
    {
       SetLastNtError(Status);
-      RETURN(FALSE);
+      Ret = FALSE;
+      goto Cleanup;
    }
 
-   RETURN(TRUE);
+   Ret = TRUE;
 
-CLEANUP:
+Cleanup:
```

Comment:
In some cases you renamed the Cleanup label to Exit, and in some cases you decided to stick with the name Cleanup. Is that difference intended anyhow? Or would it be better to structurally use the same name, e.g. Exit?

Reply:
Cleanup is used when there is something to clean up other than `UserLeave`.

Reply:
Okay, I almost anticipated that. And I am fine with that differentiation. 2 of my comments Resolved then.